library/src/commonMain/kotlin/edu/usf/cutr/otp/plan/api/PlanApi.kt

```diff
@@ -32,7 +32,7 @@ import kotlinx.serialization.json.Json
  */
 
 class PlanApi(private val url: String,
-              private val requestParameters: RequestParameters) {
+              private val requestParameters: RequestParameters? = null) {
```

Comment:
Doesn't seem like this should be nullable - you can't plan a trip without passing in parameters like origin and destination.

Reply:
I added the parameter for the instrumented tests since we don't need the origin and destination. I can't pass random values either without having to append it to the GitHub url.

Reply:
Does it hurt to append them to the GitHub URL?

I'd prefer to find another solution for testing to keep the code ideal for the production use environment (i.e. not allowing null)

Reply:
I'd still prefer to keep this as non-nullable.